privacyidea/static/components/token/controllers/tokenControllers.js

```diff
@@ -297,6 +301,25 @@ myApp.controller("tokenEnrollController", function ($scope, TokenFactory,
         $scope.setTwostepEnrollmentDefault();
     };
 
+    // helper function for setting indexed secret attribute
+    $scope.preset_indexedsecret = function() {
+        if ($scope.form.type === "indexedsecret") {
+            // Only fetch, if a preset_attribute is defined
+            if ($scope.tokensettings.indexedsecret.preset_attribute) {
+                // getUsers will only work, if we are admin
```

Comment:
do (logged in) users not have access to their attributes?

Reply:
I think you are right.
The API ``GET /users/`` can actually be called by the normal user.

Reply:
done in c0bcccb3